pootle/apps/pootle_store/receivers.py

```diff
@@ -0,0 +1,10 @@
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from pootle_store.models import Unit
+
+
+@receiver(post_save, sender=Unit)
+def unit_priority_handler(sender, **kwargs):
```

Comment:
If you do so, perhaps you can also move https://github.com/translate/pootle/blob/master/pootle/apps/virtualfolder/models.py#L459-L493 there to have all the receivers in that app in the same place.